### PR TITLE
NBB.MultiTenancy.Serilog

### PR DESCRIPTION
**Pull request by dorinsimionescu**
Serilog Enricher to put the TenantId into the LogContext

### CODE REVIEW

File: src/MultiTenancy/NBB.MultiTenancy.Serilog/TenantEventLogEnricher.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        public TenantEventLogEnricher(ITenantContextAccessor tenantContextAccessor, IOptions<TenancyHostingOptions> tenancyOptions)
+        {
+            _tenantContextAccessor = tenantContextAccessor;
+            _tenancyOptions = tenancyOptions;
+        }
+
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            if (_tenancyOptions.Value.TenancyType == TenancyType.MonoTenant || _tenantContextAccessor.TenantContext == null)
```
Comment (oncicaradupopovici) [inline src/MultiTenancy/NBB.MultiTenancy.Serilog/TenantEventLogEnricher.cs:23]: if (_tenancyOptions.Value.TenancyType == TenancyType.MonoTenant)
_tenantContextAccessor.TenantContext.GetTenantId() - already returns Tenant.Default.TenantId

When TenantContext not set you should probably set the TenantId property to null

File: src/MultiTenancy/README.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ The package [`NBB.Messaging.MultiTenancy`](../Messaging/NBB.Messaging.MultiTenan
 
 The package [`NBB.Data.EntityFramework.MultiTenancy`](../Data/NBB.Data.EntityFramework.MultiTenancy#readme) helps implementing a multi-tenant data access solution using Entity Framework.
 
+## Multi-tenant data access
```
Comment (oncicaradupopovici) [inline src/MultiTenancy/README.md:27]: wrong ttitle

**Review by oncicaradupopovici — CHANGES_REQUESTED** [review]

File: src/MultiTenancy/NBB.MultiTenancy.Serilog/TenantEventLogEnricher.cs (hunk 1 above)
Comment (dorinsimionescu) [reply src/MultiTenancy/NBB.MultiTenancy.Serilog/TenantEventLogEnricher.cs]: done

File: src/MultiTenancy/README.md (hunk 2 above)
Comment (dorinsimionescu) [reply src/MultiTenancy/README.md]: done

File: src/Tools/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Multi-tenancy
```
Comment (fraliv13) [inline src/Tools/README.md:1]: Update documentation

File: src/Tools/Serilog/NBB.Tools.Serilog.Enrichers.ServiceIdentifier/ServiceIdentifierEnricher.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        }
+
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            var source = _configuration.GetSection("Messaging")?["Source"];
+            if (string.IsNullOrEmpty(source))
+            {
+                source = Assembly.GetEntryAssembly().GetName().Name;
+            }
+            if (string.IsNullOrEmpty(source))
```
Comment (fraliv13) [inline src/Tools/Serilog/NBB.Tools.Serilog.Enrichers.ServiceIdentifier/ServiceIdentifierEnricher.cs:28]: Redundant

Comment (dorinsimionescu) [reply src/Tools/Serilog/NBB.Tools.Serilog.Enrichers.ServiceIdentifier/ServiceIdentifierEnricher.cs:28]: done

**Review by fraliv13 — APPROVED** [review]

**Pull request merged**